photon-ml/src/test/scala/com/linkedin/photon/ml/PhotonMLCmdLineParserTest.scala

```diff
@@ -65,18 +65,18 @@ class PhotonMLCmdLineParserTest {
     assertEquals(params.taskType, TaskType.LINEAR_REGRESSION)
 
     // Verify optional parameters values, should be default values
-    assertEquals(params.validateDirOpt, None)
-    assertEquals(params.maxNumIter, 80)
-    assertEquals(params.regularizationWeights, List(0.1, 1, 10, 100))
-    assertEquals(params.tolerance, 1e-6)
-    assertEquals(params.optimizerType, OptimizerType.LBFGS)
-    assertEquals(params.regularizationType, RegularizationType.L2)
-    assertEquals(params.elasticNetAlpha, None)
-    assertTrue(params.addIntercept)
-    assertTrue(params.enableOptimizationStateTracker)
-    assertFalse(params.validatePerIteration)
+    assertEquals(params.validateDirOpt, defaultParams.validateDirOpt)
+    assertEquals(params.maxNumIter, defaultParams.maxNumIter)
+    assertEquals(params.regularizationWeights, defaultParams.regularizationWeights)
+    assertEquals(params.tolerance, defaultParams.tolerance)
+    assertEquals(params.optimizerType, defaultParams.optimizerType)
+    assertEquals(params.regularizationType, defaultParams.regularizationType)
+    assertEquals(params.elasticNetAlpha, defaultParams.elasticNetAlpha)
+    assertEquals(params.addIntercept, defaultParams.addIntercept)
+    assertEquals(params.enableOptimizationStateTracker, defaultParams.enableOptimizationStateTracker)
+    assertEquals(params.validatePerIteration, defaultParams.validatePerIteration)
     assertEquals(params.minNumPartitions, 1)
-    assertTrue(params.kryo)
+    assertEquals(params.kryo, defaultParams.kryo)
     assertEquals(params.fieldsNameType, FieldNamesType.RESPONSE_PREDICTION)
     assertEquals(params.summarizationOutputDirOpt, None)
     assertEquals(params.normalizationType, NormalizationType.NONE)
```

Comment:
Why not defaultParams.minNumPartitions, defaultParams.fieldsNameType, defaultParams.normalizationType?